batcher/aligned-sdk/src/communication/messaging.rs

```diff
@@ -124,6 +127,27 @@ pub async fn receive(
             Ok(data) => data,
             Err(e) => {
                 warn!("Error while handling batcher response: {:?}", e);
+                // In the case of submitting multiple proofs we can have an multiple proofs be submitted but there
```

Comment:
These comments are a bit confusing to me

Reply:
+1